doc/COMPILE.md

```diff
@@ -606,6 +606,27 @@ running cmake like to enable debug symbol.
 For Gentoo it's recommended to emerge sys-apps/lsb-release to name the package
 right even thou not required.
 
+### With Flatpak (Beta)
+
+It is also possible to build a Flatpak package containing Elektra:
+
+```bash
+# install flatpak and flatpak-builder, using the package manager of your distro
+$ apt install flatpak flatpak-builder
+
+# download the runtime and sdk
+$ flatpak install flathub org.kde.Platform//5.15 org.kde.Sdk//5.15
+
+# build and install the package for the current user
+$ flatpak-builder .flatpak-app scripts/flatpak/org.libelektra.kdb.yaml  --force-clean --user --install
+
+# try running the kdb binary
+$ flatpak run org.libelektra.kdb
+
```

Comment:
Also show how to start qt-gui.

Btw. I get following errors/warnings:
```
Qt: Session management error: Could not open network socket
libGL error: MESA-LOADER: failed to retrieve device information
libGL error: image driver extension not found
libGL error: failed to load driver: radeon
libGL error: failed to open /dev/dri/card0: Datei oder Verzeichnis nicht gefunden
libGL error: failed to load driver: r600
qrc:/qml/WizardTemplate.qml:18:2: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/qml/main.qml:69:2: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/qml/main.qml:58:2: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
file:///usr/lib/qml/QtQuick/Controls/TextArea.qml:728:9: QML TextEdit: Cannot open: file:///usr/lib/qml/QtQuick/Controls/Base64.pdf
```

Reply:
I added the command to the README.
The warnings seem to come from a missing Radeon driver. Does the GUI start at all? I tested on a laptop with an Intel HD Graphics GPU and it worked flawlessly. My Nvidia PC, however, cannot start the GUI because of a similar error.

Reply:
Errors with Nvidia:
```
Qt: Session management error: Could not open network socket
QGLXContext: Failed to create dummy context
qrc:/qml/WizardTemplate.qml:18:2: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/qml/main.qml:69:2: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
qrc:/qml/main.qml:58:2: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
file:///usr/lib/qml/QtQuick/Controls/TextArea.qml:728:9: QML TextEdit: Cannot open: file:///usr/lib/qml/QtQuick/Controls/Base64.pdf
Failed to create OpenGL context for format QSurfaceFormat(version 2.0, options QFlags<QSurfaceFormat::FormatOption>(), depthBufferSize 24, redBufferSize -1, greenBufferSize -1, blueBufferSize -1, alphaBufferSize -1, stencilBufferSize 8, samples -1, swapBehavior QSurfaceFormat::DoubleBuffer, swapInterval 1, colorSpace QSurfaceFormat::DefaultColorSpace, profile  QSurfaceFormat::NoProfile
```

Reply:
> Does the GUI start at all?

Yes, it started. These warnings/errors look like they are beyond of the scope of this PR. The "Base64.pdf" looks like a file which was not installed. The path /usr/lib/qml/QtQuick/Controls doesn't make sense, though. I don't think we should install it there.

Reply:
I managed to get this to run on an Nvidia GPU, using a quickfix from here:

https://github.com/flathub/com.calibre_ebook.calibre/issues/120

TL;DR:

```bash
export QT_XCB_GL_INTEGRATION=none
```